docker_daemon/check.py

```diff
@@ -637,14 +638,46 @@ def _report_cgroup_metrics(self, container, tags):
                         if value is not None:
                             metric_func(self, mname, value, tags=tags)
 
+    def _build_network_mapping(self, container):
+        """Matches /proc/$PID/net/route and docker inspect to map interface names to docker network name"""
+        try:
+            proc_net_route_file = os.path.join(container['_proc_root'], 'net/route')
```

Comment:
You're missing the `pid` (`/proc/<pid>/net/route`) here in case it's in a different network namespace.